internal/venice-common/src/main/java/com/linkedin/venice/meta/ZKStore.java

```diff
@@ -448,6 +447,8 @@ public void setHybridStoreConfig(HybridStoreConfig hybridStoreConfig) {
     } else {
       this.storeProperties.hybridConfig = hybridStoreConfig.dataModel();
     }
+    // for backward compatibility
+    this.storeProperties.incrementalPushEnabled = this.storeProperties.hybridConfig != null;
```

Comment:
This diff is too big so I would like to double confirm on the intention:
I can see that you want to remove the inc push enabled flag and make inc push available when hybrid is enabled.
This means we should change all places where incPushEnabled flag is used to check on hybrid enabled.
Once this change is deployed to prod, we can cut a cleanup PR to remove this field from protocol.
Is that right?

Reply:
Yes we want to remove `inc-push stores` concept.

> This means we should change all places where incPushEnabled flag is used to check on hybrid enabled.

Yes, that's correct. But